rubric_visualization/settings/local.py

```diff
@@ -5,6 +5,8 @@
 
 INSTALLED_APPS.append('sslserver')
 
+WEBPACK_DEV = True
```

Comment:
Suggestion: maybe add a comment to indicate what this does, since it's a custom setting we've added.

Also, does this have a default value in `base.py`? Otherwise, I think an `AttributeError` will be raised when using one of the other django settings modules (I could be wrong).